ui/app/AppLayouts/Chat/views/ChatContentView.qml

```diff
@@ -381,10 +389,10 @@ ColumnLayout {
                         msg = chatInput.interpretMessage(msg)
 
                         chatContentModule.inputAreaModule.sendMessage(
-                            msg,
-                            chatInput.isReply ? SelectedMessage.messageId : "",
-                            Utils.isOnlyEmoji(msg) ? Constants.messageContentType.emojiType : Constants.messageContentType.messageType,
-                            false)
+                                    msg,
+                                    chatInput.isReply ? chatInput.replayMessageId : "",
```

Comment:
Not sure if this is new typo or an old one, but we should rename it all to `replyMessageId` instead of `replay...`